src/Pages/Authentication/ForgotPassword/ForgotPassword.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import '../Css/Authentication.css';
+import Input from '../../../Shared/Input/Input';
+import blitheCollage from '../../../Assets/Images/Blithe-collage.jpg';
+import blitheLogo from '../../../Assets/Images/Blithe-logo.png';
+import InputField from '../../../Shared/Input/InputField';
+
+const ForgotPassword = () =>{
+
+    const [Data, setData] = useState(
+        {   
+            email:'',
+        })
+
+    const handleSubmit = (e) =>{
+        e.preventDefault();
+        //commented right now for future perspective..
+        console.log('Data', Data);
```

Comment:
don't leave console.log in code

Reply:
/commented right now for future perspective..